### ISSUE TITLE
windows 10 build 1903 kills Ps move service ?

### ISSUE THREAD

**Issue by FanTaz1986** [HipsterSloth/PSMoveSteamVRBridge#90]
so its only me or 1903 kills ps move service ? 
i cant pair controlers on this windows version
does any one was able to do pair ?
if not, how much posible is to fix to come or its not posible anymore ?
june 11 did some BT "fixes"

**Comment by Zentrik** [comment]
I could not get controllers to pair with 1903 on a fresh installation getting verification failed, downgrading to 1703 fixed this

**Comment by FanTaz1986** [comment]
i did downgrade to 1809 and its works, so ir for sure 1903 problems
so windows fix it, or new way to pair must by found
worst part is 1903 is realy good pach its give nice ryzen boost and other nice stuff

**Comment by Yozer** [comment]
I had the same issue. Solution:
https://github.com/psmoveservice/PSMoveService/issues/631#issuecomment-515784561

**Comment by HonkeyKong** [comment]
I don't know if it'll work for everyone, but the workaround I found was installing the Toshiba Bluetooth stack over the Windows BT stack. Once I did that and rebooted my PC, everything worked just like it did before KB4503293. Now if only I could get SteamVR to stop crashing when I load it, I'd be golden.

You can download the Toshiba stack for Windows here: https://www.softpedia.com/get/Internet/Remote-Utils/Toshiba-Bluetooth-Stack.shtml

**Comment by manobroda** [comment]
I founded a solution by another user that maked a custom version of psmoveserviceadmin for Windows 10. It was hard, but I got to pair my psmove controllers with this mod and with a trick to remove the devide in windows 10 while pairing.  Not easy at all!

**Comment by Dvd97** [comment]
> Encontré una solución de otro usuario que creó una versión personalizada de psmoveserviceadmin para Windows 10. Fue difícil, pero pude emparejar mis controladores psmove con este mod y con un truco para eliminar el dispositivo en Windows 10 durante el emparejamiento. ¡No es nada fácil!

Me podrías facilitar la versión personalizada que estas hablando

**Comment by GamerHun1238** [comment]
yes but it can be fixed when psmoveservice api updates to the latest version.

**Comment by GamerHun1238** [comment]
> I founded a solution by another user that maked a custom version of psmoveserviceadmin for Windows 10. It was hard, but I got to pair my psmove controllers with this mod and with a trick to remove the devide in windows 10 while pairing. Not easy at all!

whats the user's name?

**Comment by omran3214** [comment]
> I had the same issue. Solution:
> [psmoveservice/PSMoveService#631 (comment)](https://github.com/psmoveservice/PSMoveService/issues/631#issuecomment-515784561)

Hi the download link isnt working!
and i dont know how to build a source code (im a newbie lol)
could you send me your build?

**Comment by jasonhillpd276** [comment]
I have been struggling with this too. Has there been any updates on a new x64 build with installer?
